pdns/recursordist/docs/settings.rst

```diff
@@ -252,6 +252,15 @@ Which domains we only accept delegations from (a Verisign special).
 Turn off the packet cache. Useful when running with Lua scripts that can
 not be cached.
 
+.. _setting-disable-real-memory-usage:
```

Comment:
```suggestion
.. _setting-disable-real-memory-usage:
.. versionadded:: 4.1.13
```

Reply:
what's the suggested change on the `.. _setting-disable-real-memory-usage:` line? It is marked as a change, but I cannot set the difference...

Reply:
There is none, no idea why GH displays it that way.